core/src/main/java/com/microsoft/applicationinsights/internal/heartbeat/DefaultHeartBeatPropertyProvider.java

```diff
@@ -1,5 +1,8 @@
 package com.microsoft.applicationinsights.internal.heartbeat;
 
+import javax.annotation.Nonnull;
```

Comment:
Should I use [`@org.checkerframework.checker.nullness.qual.NonNull`](https://checkerframework.org/api/org/checkerframework/checker/nullness/qual/NonNull.html) instead?

Reply:
This can be removed.